diff --git a/nspr/configure.in b/nspr/configure.in
index 112ddd2..7b4fcae 100644
--- a/nspr/configure.in
+++ b/nspr/configure.in
@@ -1098,12 +1098,6 @@ no)
     thumb_flag="-marm"
     ;;
 *)
-    _SAVE_CFLAGS="$CFLAGS"
-    CFLAGS="$arch_flag"
-    AC_TRY_COMPILE([],[return sizeof(__thumb2__);],
-        MOZ_THUMB2=1,
-        MOZ_THUMB2=)
-    CFLAGS="$_SAVE_CFLAGS"
     thumb_flag=""
     ;;
 esac
