diff --git a/mozilla/nsprpub/config/nspr-config.in b/mozilla/nsprpub/config/nspr-config.in
index a7d2b4f..2f441be 100755
--- a/mozilla/nsprpub/config/nspr-config.in
+++ b/mozilla/nsprpub/config/nspr-config.in
@@ -98,7 +98,7 @@ if test -z "$includedir"; then
     includedir=@includedir@
 fi
 if test -z "$libdir"; then
-    libdir=@libdir@
+    libdir=`pkg-config --variable=libdir nspr`
 fi
 
 if test "$echo_prefix" = "yes"; then
@@ -132,12 +132,12 @@ if test "$echo_libs" = "yes"; then
       if test -n "$lib_nspr"; then
 	libdirs="$libdirs -lnspr${major_version}"
       fi
-      os_ldflags="@LDFLAGS@"
+      os_ldflags=`pkg-config --variable=ldflags nspr`
       for i in $os_ldflags ; do
 	if echo $i | grep \^-L >/dev/null; then
 	  libdirs="$libdirs $i"
         fi
       done
-      echo $libdirs @OS_LIBS@
+      echo $libdirs `pkg-config --variable=os_libs nspr`
 fi      
 
